satellite/satellitedb/coinpaymentstxs.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+	"math/big"
+
+	"storj.io/storj/satellite/payments/coinpayments"
+	"storj.io/storj/satellite/payments/stripecoinpayments"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+// coinpaymentsTransactions is Coinpayments transactions DB.
```

Comment:
```suggestion
// coinpaymentsTransactions is Coinpayments transactions DB.
//
// architecture: Database
```

You also have to add to this file the following

```var _ TransactionsDB = (*TransactionsDB)(nil)```

Reference https://storj.slack.com/archives/CAQV0AE2Z/p1571246875383300